src/Java/grundlagen/j6/annotations/DocumentationProcessor.java

```diff
@@ -1,37 +0,0 @@
-package Java.grundlagen.j6.annotations;
-
-import javax.annotation.processing.AbstractProcessor;
-import javax.annotation.processing.RoundEnvironment;
-import javax.annotation.processing.SupportedAnnotationTypes;
-import javax.annotation.processing.SupportedSourceVersion;
-import javax.lang.model.SourceVersion;
-import javax.lang.model.element.Element;
-import javax.lang.model.element.TypeElement;
-import java.util.Arrays;
-import java.util.Set;
-
-@SupportedAnnotationTypes({"Java.grundlagen.j6.annotations.Excercise1"})
-@SupportedSourceVersion(SourceVersion.RELEASE_8)
-public class DocumentationProcessor extends AbstractProcessor {
-
-    @Override
-    public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
-        try {
-            for (TypeElement annotation : annotations) {
-                Set<? extends Element> annotatedElements = roundEnv.getElementsAnnotatedWith(annotation);
-                for (Element element : annotatedElements) {
-                    Excercise1 annotation1 = element.getAnnotation(Excercise1.class);
-                    Class<?> clazz = Class.forName("Java.grundlagen.j6.annotations.Excercise2");
```

Comment:
Wieso gehst du hier über einen String?

```suggestion
Class<?> clazz = Excercise2.class;
```

Ausserdem brauchst du diese Variable gar nicht wirklich, oder? Bitte die Referenz entfernen, damit die Model-Klasse nicht dazu kompiliert werden muss. Du willst ja via AnnotationsProzessor irgendeine Java-Klasse angeben können, die noch nicht kompiliiert ist.